resources/core/charts/apiserver-proxy/templates/deployment.yaml

```diff
@@ -53,7 +53,9 @@ spec:
         - --v=10
         - --oidc-issuer=https://dex.{{ .Values.global.ingress.domainName }}
         - --oidc-clientID=kyma-client
+        {{ if not .Values.global.environment.gardener }}
```

Comment:
Suggestion: If we are doing multiple ifs that all depend on the same condition (isGardener here), it may be better to have 2 depoyment files (deployment and deployment-gardener) and pick which one we use.